tools/accuracy_checker/configs/mosaic.yml

```diff
@@ -0,0 +1,35 @@
+models:
+  - name: style_transfer
+
+    launchers:
+
+      - framework: onnx_runtime
+        device: CPU
+        model: public/mosaic/mosaic.onnx
+        adapter: style_transfer
+
+      - framework: dlsdk
+        device: CPU
+        model: public/mosaic/FP32/mosaic.xml
+        weights: public/mosaic/FP32/mosaic.bin
+        adapter: style_transfer
+
+    datasets:
+      - name: coco_style_transfer
+        data_source: val2017
+        annotation_conversion:
+          converter: style_transfer
+          images_dir: val2017
+          dst_height: 224
+          dst_width: 224
```

Comment:
please move data source and annotation conversion to dataset_definitions

Reply:
Fixed